tests/Core/Config/SniffsExcludeArgsTest.php

```diff
@@ -144,15 +189,50 @@ public static function dataValidSniffs()
         $data      = [];
 
         foreach ($arguments as $argument) {
```

Comment:
As the data provider format has changed, the `@return` for the method needs to be updated.